deploy/docker/compose.local.secure.yml

```diff
@@ -0,0 +1,61 @@
+services:
+  traefik:
+    image: traefik:v2.10
+    restart: unless-stopped
+    command: 
+      - "--log.level=DEBUG"
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      - "--entryPoints.web.address=:80"
+      - "--entrypoints.web-secure.address=:443"
+      - "--entrypoints.web.http.redirections.entryPoint.to=web-secure"
+      - "--entrypoints.web.http.redirections.entryPoint.scheme=https"
+      - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
+      - "--providers.file.directory=/etc/traefik/dynamic"
+      - "--providers.file.watch=true"
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - "${DTAAS_DIR}/deploy/docker/dynamic/tls.local.yml:/etc/traefik/dynamic/tls.yml"
```

Comment:
@prasadtalasila Since we've removed forward auth from the localhost setup, I believe we don't need to supply a TLS certificate to run the application over https.

I've verified this by removing this line and deleting the certificates on my machine, and it works fine. Should I remove this line along with the certificates directory and `tls.local.yml`?